src/app/helpers.js

```diff
@@ -240,6 +240,15 @@ function getErrorMessageForRelayModernProblem(errorOrErrors) {
   return null;
 }
 
+function getErrorObjectForRelayModernProblem(errorOrErrors) {
```

Comment:
Please make it `getErrorObjectsForRelayModernProblem` for consistency.

Reply:
Done